hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -356,6 +357,10 @@ public void initialize() throws HBaseIOException {
     // Create the balancer
     Class<? extends LoadBalancer> balancerKlass = config.getClass(HBASE_RSGROUP_LOADBALANCER_CLASS,
         StochasticLoadBalancer.class, LoadBalancer.class);
+    if (balancerKlass.equals(this.getClass())) {
+      LOG.warn("The internal balancer of RSGroupBasedLoadBalancer cannot be itself, falling back to the default LoadBalancer class");
```

Comment:
line len > 100 chars

Reply:
@virajjasani Thanks for review, I resubmitted a PR for branch-2.2